Content.Server/Chemistry/Components/ReagentDispenserComponent.cs

```diff
@@ -351,30 +365,27 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs args)
 
             var solutionSys = EntitySystem.Get<SolutionContainerSystem>();
             var activeHandEntity = hands.GetActiveHand.Owner;
-            if (solutionSys.TryGetSolution(activeHandEntity, "beaker", out _))
+            if (HasBeaker)
```

Comment:
Again, this belongs in a system. Not Component. We're abandoning Component-based logic.

Reply:
I feel like this is the job for some sort of reagent dispenser refactoring, not a bugfix.

Reply:
If the component is not ECS'd yet, don't force people to ECS it for bugfixes like this, Y.

Reply:
Pretty sure I was asked to bugfix and refactor various stuff unrelated to solution in solution refactor. E.g. Grindable. 

Additionally, this is just more work for me, while I refactor Chem machines.

Reply:
Given the reduction in copy/pasted code, I doubt it would add more work.

Reply:
This is fine.